compiler/rustc_interface/src/passes.rs

```diff
@@ -979,6 +979,10 @@ fn analysis(tcx: TyCtxt<'_>, (): ()) -> Result<()> {
 
     sess.time("layout_testing", || layout_test::test_layout(tcx));
 
+    sess.time("stable_impl_const_trait_checking", || {
+        rustc_passes::stability::check_const_impl_trait(tcx)
+    });
```

Comment:
Where should this statement go? If it's any lower, then it won't run at the "correct" time.

Reply:
If possible it should be done in `check_mod_unstable_api_usage`, if not, then call it next to the invocation further up.

Reply:
Do you mean:

1. move the call to `check_const_impl_trait` to that function, or
2. move the code from `impl ItemLikeVisitor for CheckStableConstImplTrait` to `impl Visitor for Checker`?

I can see both sides of this. On one hand, the former would be easier to remove once `impl const Trait` is stabilized. But on the other hand, the latter would be more efficient, I would presume.